hack/sync-kubevirtci.sh

```diff
@@ -10,7 +10,9 @@ version_file="cluster-up/version.txt"
 sha_file="cluster-up-sha.txt"
 download_cluster_up=true
 function getClusterUpShasum() {
-    find ${KUBEVIRT_DIR}/cluster-up -type f | sort | xargs sha1sum | sha1sum | awk '{print $1}'
+    pushd ${KUBEVIRT_DIR} >/dev/null
```

Comment:
you can do 
```bash
(
    cd $KUBIVRT_DIR
    find ...
)
```

Parentesis are creating a subshell and after the subshell you will be at the previous dir.

Reply:
done thx